src/docker.rs

```diff
@@ -0,0 +1,53 @@
+/// Module that pretends to delete Docker images
+use rand::prelude::*;
+use rand::Rng;
+
+use crate::utils::csleep;
+use crate::utils::gen_hex_string;
+use crate::DOCKER_PACKAGES_LIST;
+use crate::DOCKER_TAGS_LIST;
+use crate::parse_args::AppConfig;
+
+
+pub fn run(appconfig: &AppConfig) {
+    let mut rng = thread_rng();
+    let num_packages = rng.gen_range(20, 100);
+    // Choose `num_packages` packages, non-repeating and in random order
+    let chosen_packages: Vec<_> = DOCKER_PACKAGES_LIST.choose_multiple(&mut rng, num_packages).collect();
+
+    for _stage in &["Deleting"] {
```

Comment:
What is this look supposed to do? It's looking over a list with one element. Seems like a no-op to me. Maybe some old left-over code?